src/corona_hakab_model/medical_state_consts/medical_state_machine_creator.py

```diff
@@ -0,0 +1,110 @@
+import json
+import numbers
+from typing import Dict, List, TypedDict, Union, Optional
+
+from errors import InvalidJSON
+from medical_state import StochasticMedicalState, TerminalMedicalState, MedicalState
+from medical_state_machine import MedicalStateMachine
+from state_machine import State
+from util import dist
+
+
+class ProbabilityData(TypedDict):
+    type: str
+    from_state: str
+    to: str
+
+
+class MedicalStateData(TypedDict, total=False):
+    susceptible: bool
+    contagiousness: float
+    test_willingness: float
+    detectable: bool
+    name: str
+    stochastic: Optional[bool]
+    terminal: Optional[bool]
+
+
+class StateTransitionData(TypedDict):
+    from_state: str
+    to: str
+    distribution: List[float]
+    probability: Union[float, ProbabilityData]
+
+
+class MedicalStateMachineData(TypedDict):
+    medical_states: Dict[str, MedicalStateData]
+    initial_state: str
+    default_state_upon_infection: str
+    medical_state_transitions: List[StateTransitionData]
+    sick_states: List[str]
+    was_ever_sick_states: List[str]
+
+
+class MedicalStateMachineCreator:
```

Comment:
Personally I prefer it be called MedicalStateMachineBuilder since it has zero "creative" input inn the creation, it only assembles a structure from a file